mod_reforged/hooks/skills/racial/alp_racial.nut

```diff
@@ -1,4 +1,45 @@
 ::mods_hookExactClass("skills/racial/alp_racial", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.Name = "Alp";
+		this.m.Icon = "ui/orientation/alp_01_orientation.png";
+		this.m.IsHidden = false;
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/melee_defense.png",
+				text = "Reduces damage taken from melee piercing attacks by " + ::MSU.Text.colorRed("50%")
+			},
+			{
+				id = 11,
+				type = "text",
+                icon = "ui/icons/ranged_defense.png",
+				text = "Reduces damage taken from ranged piercing and blunt attacks by " + ::MSU.Text.colorRed("66%")
+			},
+			{
+				id = 12,
+				type = "text",
+                icon = "ui/icons/melee_defense.png",
+				text = "Reduces damage taken from dog and wolf cutting attacks by " + ::MSU.Text.colorRed("66%")
```

Comment:
66% reduced damage received from dog and wolf cutting attacks

Reply:
I went with
`66% reduced cutting damage received from dogs and wolfs`

Reply:
Ok that is fine too. But change it to `wolves` as that is the correct plural word for wolf.